### ISSUE TITLE
[FIX]: Accessible header

### ISSUE THREAD

**Issue by julesbruins** [fdnd-agency/buurtcampuskrant#70]
# Description
Make sure the header is tested on accessibility. Tabbing doesn't work like it should on the header.

**Comment by julesbruins** [comment]
<img width="913" height="302" alt="Image" src="https://github.com/user-attachments/assets/163447b8-b1d7-4a14-b91a-67de5779adbd" />

Fixed tabbing on the header. The rest was perfect!

<img width="261" height="129" alt="Image" src="https://github.com/user-attachments/assets/05f55bdc-1deb-41a4-a8ae-b66f16c00a10" />
